mslib/mscolab/sockets_manager.py

```diff
@@ -283,6 +326,7 @@ def setup_managers(app):
     socketio.on_event('file-save', sm.handle_file_save)
     socketio.on_event('add-user-to-operation', sm.join_creator_to_operation)
     socketio.on_event('update-operation-list', sm.update_operation_list)
+    socketio.on_event('operation-selected', sm.handle_operation_selected)  # Register new event
```

Comment:
the comment is no new informations, thats what happens by the line.  you may want to comment for what it is used. We later can do with the others too.